kuma/wiki/models.py

```diff
@@ -1484,6 +1485,14 @@ def has_legacy_namespace(self):
         namespace, separator, _ = self.slug.partition(":")
         return namespace in LEGACY_MINDTOUCH_NAMESPACES if separator else False
 
+    @property
+    def has_noindex_slug(self):
+        """Return true if the slug is in hand-picked list of prefixes."""
+        for slug_start in NOINDEX_SLUG_STARTS:
+            if self.slug.startswith(slug_start):
+                return True
+        return False
```

Comment:
I really want to code golf that down to:

```python
@property
def has_noindex_slug(self):
    """Return true if the slug is in hand-picked list of prefixes."""
    return any(self.slug.startswith(slug_start) for slug_start in NOINDEX_SLUG_STARTS)
```

...but this is Python, and so that's probably a bad idea. :)

Reply:
I'm not even convinced that's better that the "long form". It's "hipper" to use `any()` but I think it reads well as a "dumb" for loop.